### ISSUE TITLE
Add webcrypto polyfill to gh-pages

### ISSUE THREAD

**Issue by rmhrisk** [PeculiarVentures/xadesjs#27]
For the page to work on safari we will probably need to add a polyfill

**Comment by rmhrisk** [comment]
https://github.com/vibornoff/webcrypto-shim

**Comment by rmhrisk** [comment]
Added the shim, was not enough to make work on Safari, blocked on https://github.com/PeculiarVentures/PKI.js/issues/7

**Comment by rmhrisk** [comment]
Bug was to add polyfill that was done. Will create another issue to track the root cause.
